gradle/shared-settings/build.gradle.kts

```diff
@@ -0,0 +1,7 @@
+plugins {
+    `kotlin-dsl`
+}
+
+dependencies {
+    implementation("org.gradle.toolchains:foojay-resolver:0.5.0") // https://github.com/gradle/foojay-toolchains
```

Comment:
All dependencies should be declared in the TOML file, to allow Dependabot to pick them up for automatic updates.

Reply:
Alright. I pushed another commit.